Source/Libraries/NetCore/DebugInfo/CloudDebug_Form.cs

```diff
@@ -344,7 +344,7 @@ internal static class CloudTransfer
     {
         private static string CorruptCloudServer = "http://cc.r5x.cc/rtc/debug";
 
-        public static string CloudLoad(string filename, string password)
```

Comment:
This also seems like it could be a bug. I'm not sure if the file is supposed to be protected by a password on the server, or if the file itself is just encrypted with a password.

Reply:
The file is just encrypted with a password. I think at one point it was going to be protected on the server, but it was changed to just file encryption. I don't remember the exact design reason for this, but I think the goal was so we could make a relatively simple error reporting system where the files weren't accessible unless we were explicitly given access.